Author: Alastair McKinstry <mckinstry@debian.org>
Description: Add explicit declaration.
Last-Updated: 2014-10-28
Forwarded: no

Index: cdo-1.6.5+dfsg.1/src/griddes.c
===================================================================
--- cdo-1.6.5+dfsg.1.orig/src/griddes.c
+++ cdo-1.6.5+dfsg.1/src/griddes.c
@@ -55,6 +55,7 @@ void extClose(int fileID);
 
 #define MAX_LINE_LEN 65536
 
+void str2uuid(const char *uuidstr, char *uuid);
 
 void gridInit(griddes_t *grid)
 {
